tests/TestContext.cs

```diff
@@ -1,437 +1,441 @@
-
-            token = stdout.Trim();
-        }
-
-        if (string.IsNullOrEmpty(token))
-        {
-            throw new Exception("GitHub token not found, run `gh auth login` to authenticate");
-        }
-
-        return token;
-    }
-
-    private static async Task<OldGitHubClient> GetOldGitHubClient(ITestOutputHelper outputHelper)
-    {
-        if (_sharedOldGitHubClient != null)
-        {
-            return _sharedOldGitHubClient;
-        }
-
-        var token = await GetGitHubToken(outputHelper);
-        _sharedOldGitHubClient = new OldGitHubClient(new ProductHeaderValue("renovate-test"), new InMemoryCredentialStore(new Octokit.Credentials(token)));
-
-        return _sharedOldGitHubClient;
-    }
-
-    private static async Task<NewGitHubClient> GetNewGitHubClient(ITestOutputHelper outputHelper)
-    {
-        if (_sharedNewGitHubClient != null)
-        {
-            return _sharedNewGitHubClient;
-        }
-
-        var token = await GetGitHubToken(outputHelper);
-        var tokenProvider = new TokenProvider(token);
-        var adapter = RequestAdapter.Create(new TokenAuthProvider(tokenProvider));
-        _sharedNewGitHubClient = new NewGitHubClient(adapter);
-
-        return _sharedNewGitHubClient;
-    }
-
-    private static void CopyRenovateFile(TemporaryDirectory temporaryDirectory)
-    {
-        var gitRoot = GetGitRoot();
-
-        File.Copy(gitRoot / "default.json", temporaryDirectory.FullPath / "renovate.json");
-    }
-
-    private static FullPath GetGitRoot()
-    {
-        if (FullPath.CurrentDirectory().TryFindFirstAncestorOrSelf(current => Directory.Exists(current / ".git"), out var root))
-        {
-            return root;
-        }
-
-        throw new InvalidOperationException("root folder not found");
-    }
-
-    private static async Task<(string StdOut, string StdError)> ExecuteCommand(ITestOutputHelper outputHelper,
-        string executable, string[] args, KeyValuePair<string, string>[]? envVariables = null)
-    {
-        var stdOut = new StringBuilder();
-        var stdErr = new StringBuilder();
-        await Cli.Wrap(executable).WithArguments(args)
-            .WithEnvironmentVariables(builder =>
-            {
-                foreach (var (key, value) in envVariables ?? [])
-                {
-                    builder.Set(key, value);
-                }
-            })
-            .WithStandardOutputPipe(PipeTarget.Merge(PipeTarget.ToStringBuilder(stdOut), PipeTarget.ToDelegate(outputHelper.WriteLine)))
-            .WithStandardErrorPipe(PipeTarget.Merge(PipeTarget.ToStringBuilder(stdErr), PipeTarget.ToDelegate(outputHelper.WriteLine)))
-            .ExecuteAsync();
-
-        return (stdOut.ToString(), stdErr.ToString());
-    }
-
-    public async ValueTask DisposeAsync()
-    {
-        await repositoryDirectory.DisposeAsync();
-        await this.CleanupRepository();
-    }
+using System.Diagnostics.CodeAnalysis;

```

Comment:
The web diff is super weird. I used Beyond Compare to look at the actual changes, which looks good, although filtering out the initial commit message doesn't seem to have any effect?

Reply:
Oops just realized this is a draft

Reply:
Not meant to be merged. I'll update the title

Reply:
I think encoding or eol changed

Reply:
Much better https://github.com/workleap/renovate-config/pull/76/files?diff=split&w=1